README.md

```diff
@@ -1,7 +1,8 @@
 # README 
 [![Maintainability](https://api.codeclimate.com/v1/badges/ce8783c839063c27b944/maintainability)](https://codeclimate.com/github/Fermented-Software/Kimchi/maintainability)
 [![Coverage Status](https://coveralls.io/repos/github/egydiopacheco/Kimchi/badge.svg?branch=main)](https://coveralls.io/github/egydiopacheco/Kimchi?branch=main)
-
+![GA Release Workflow main](https://github.com/egydiopacheco/Kimchi/actions/workflows/ci-release.yaml/badge.svg?branch=main)
+![GA Pull Request Workflow main](https://github.com/egydiopacheco/Kimchi/actions/workflows/ci-release.yaml/badge.svg?branch=main)
```

Comment:
Both badges have the same URL. Why is that?

Reply:
True, I need to change it. But before commiting the change, I noticed that for PR yaml file the status is returning: _no status_ instead of _passing_ for some reason

Reply:
One can only wonder....

Reply:
Are you sure you need to specify a branch for the badge?

Reply:
If so, then it's because there's no PR opened to main branch

Reply:
I think we need to specify the main branch, and the status needs to be _passing_ according to the specification
<img width="515" alt="image" src="https://user-images.githubusercontent.com/66056818/191357747-f786215f-bb87-42d4-ae08-173628cf49c8.png">

Reply:
Well, it's unclear if he wants both the Release Status and the PR status. Since we won't always have a PR to main, I think it's better if we keep just the release badge.

What do you think? Let's involve others as well in this conversation.
@egydiopacheco @lk-vila @raulmel1o

Reply:
I also find that unclear, in that case, I think that the best approach would be to keep both. We also can send a message to Daniel or Miguel to make it clearer.

Reply:
So let's keep it and then talk to the prof.